.gitignore

```diff
@@ -74,12 +74,4 @@ yarn-debug.log*
 /artifacts/
 
 # Ignore solr files
-/solr/geodata-test/data/index/*
-/solr/geodata-test/data/tlog/*
-/solr/geodata/data/index/*
-/solr/geodata/data/tlog/*
-
-!/solr/geodata-test/data/index/.keep
-!/solr/geodata-test/data/tlog/.keep
-!/solr/geodata/data/index/.keep
-!/solr/geodata/data/tlog/.keep
+solr/*/data/*
```

Comment:
@yzhoubk This is why you've run into Solr problems when starting the dev stack. The data directories should be completely omitted—Solr will create them automatically on startup. If starting from scratch, run the `geoblacklight:index:seed` task to seed Solr with fixture data. If you've previously started the stack, you should be able to restart it on top of your existing data directories without issue.

Reply:
@danschmidt5189 Yes, I found it was caused by the `.keep` files under /data/tlog and data/index. Sure, let's skip subdirectories under data/.  Thanks